.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           tests/resources/dockerursim/build_and_run_docker_ursim.sh $URSIM_VERSION
         env: ${{matrix.env}}
       - name: install gtest
-        run: sudo apt-get install -y libgtest-dev
+        run: sudo apt-get install -y libgtest-dev libboost-all-dev
       - name: configure
         run: mkdir build && cd build && cmake .. -DBoost_INCLUDE_DIR=$BOOST_ROOT_1_72_0/include -DBUILDING_TESTS=1 -DINTEGRATION_TESTS=1
```

Comment:
I'm not too sure about details here, but why the specific version of boost?
Is [our fix](https://github.com/UniversalRobots/Universal_Robots_Client_Library/pull/37/commits/c5de0327b99347b072a8d5fe60289111cb369f57) similar, or does that change something unexpected?

Reply:
Yes, that should be checked, whether it can be removed.

Reply:
Updated the PR accordingly.